atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/InternalLockWatchManager.java

```diff
@@ -32,4 +34,6 @@
      */
     TransactionsLockWatchEvents getEventsForTransactions(Set<Long> startTimestamps,
             Optional<IdentifiedVersion> lastKnownVersion);
+
+    CommitUpdate getCommitUpdate(long startTimestamp, LockToken commitLocksToken);
```

Comment:
I'm kind of surprised that we'd need an extra type for this. Ideally I'd expect that the methods on this class be:

```java
IdentifiedVersion getVersion(long timestamp);
LockWatchEvents getLockWatchUpdates(Optional<IdentifiedVersion> from, Optional<IdentifiedVersion> to);
```